src/Microsoft.Framework.PackageManager/Install/InstallGlobalCommand.cs

```diff
@@ -67,6 +67,20 @@ public FeedOptions FeedOptions
 
             var temporaryProjectFileFullPath = CreateTemporaryProject(FeedOptions.TargetPackagesFolder, packageId, packageVersion);
 
+            var packageFullPath = Path.Combine(
+                _commandsRepository.PackagesRoot.Root,
+                _commandsRepository.PathResolver.GetPackageDirectory(packageId, new SemanticVersion(packageVersion)));
+
+            if (OverwriteCommands)
+            {
+                if (Directory.Exists(packageFullPath))
+                {
+                    WriteVerbose($"Deleting {packageFullPath}");
+                    FileOperationUtils.DeleteFolder(packageFullPath);
+                    Directory.Delete(packageFullPath);
```

Comment:
I don't think we need to catch it. If it ever happens then we shouldn't continue the global install and the exception has as much information in it as we know.